ci/build_windows.py

```diff
@@ -115,6 +117,7 @@ class BuildFlavour(Enum):
         '-DCUDA_ARCH_NAME=Manual '
         '-DCUDA_ARCH_BIN=52 '
         '-DCUDA_ARCH_PTX=52 '
+        '-DCUDA_ARCH_LIST=5.2 '
```

Comment:
Why add `CUDA_ARCH_LIST`?

Reply:
the first class cuda use CUDA_ARCH_LIST choose or set arch

Reply:
I don't think it works currently. I got this warning when testing it:

```
Manually-specified variables were not used by the project:

    CUDA_ARCH_LIST
```

It is fixed as part of https://github.com/apache/incubator-mxnet/pull/17031

Reply:
currently works  fine on ci.The warning may be caused by other modifications.
![image](https://user-images.githubusercontent.com/771744/71071314-56f74980-21b7-11ea-840f-7203f1655967.png)

Reply:
I just tried it again. It only works with `CUDA_ARCH_LIST=5.2` but not with `CUDA_ARCH_LIST=7.0`. It seems somewhere an outdated arch detection script is used.

Reply:
yes.only in https://github.com/apache/incubator-mxnet/pull/16980/files#diff-101d74caab6e808c727c01a427bd927eR137